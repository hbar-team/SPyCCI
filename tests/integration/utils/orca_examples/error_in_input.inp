%pal
  nprocs 8
end

%maxcore 750

! PBU def2-SVP 
! RIJCOSX def2/J


* xyzfile 0 1 water.xyz

