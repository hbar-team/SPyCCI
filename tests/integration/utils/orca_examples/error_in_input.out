
                                 *****************
                                 * O   R   C   A *
                                 *****************

                                            #,                                       
                                            ###                                      
                                            ####                                     
                                            #####                                    
                                            ######                                   
                                           ########,                                 
                                     ,,################,,,,,                         
                               ,,#################################,,                 
                          ,,##########################################,,             
                       ,#########################################, ''#####,          
                    ,#############################################,,   '####,        
                  ,##################################################,,,,####,       
                ,###########''''           ''''###############################       
              ,#####''   ,,,,##########,,,,          '''####'''          '####       
            ,##' ,,,,###########################,,,                        '##       
           ' ,,###''''                  '''############,,,                           
         ,,##''                                '''############,,,,        ,,,,,,###''
      ,#''                                            '''#######################'''  
     '                                                          ''''####''''         
             ,#######,   #######,   ,#######,      ##                                
            ,#'     '#,  ##    ##  ,#'     '#,    #''#        ,####,    ,#,          
            ##       ##  ##   ,#'  ##            #'  '#       #'       ,# #          
            ##       ##  #######   ##           ,######,      #####,      #          
            '#,     ,#'  ##    ##  '#,     ,#' ,#      #,     #,   #      #          
             '#######'   ##     ##  '#######'  #'      '#     '####' #    #          



                #########################################################
                #                        -***-                          #
                #          Department of theory and spectroscopy        #
                #                                                       #
                #                      Frank Neese                      #
                #                                                       #
                #     Directorship, Architecture, Infrastructure        #
                #                    SHARK, DRIVERS                     #
                #        Core code/Algorithms in most modules           #
                #                                                       #
                #        Max Planck Institute fuer Kohlenforschung      #
                #                Kaiser Wilhelm Platz 1                 #
                #                 D-45470 Muelheim/Ruhr                 #
                #                      Germany                          #
                #                                                       #
                #                  All rights reserved                  #
                #                        -***-                          #
                #########################################################


                         Program Version 6.1.0  -  RELEASE   -
                                (GIT: $679e74b$)
                          ($2025-06-10 18:02:51 +0200$)


 With contributions from (in alphabetic order):
[Max-Planck-Institut fuer Kohlenforschung]                               
  Daniel Aravena         : Magnetic Suceptibility                        
  Michael Atanasov       : Ab Initio Ligand Field Theory (pilot matlab implementation)
  Alexander A. Auer      : GIAO ZORA, VPT2 properties, NMR spectrum      
  Ute Becker             : All parallelization in ORCA, NUMFREQ, NUMCALC 
  Giovanni Bistoni       : ED, misc. LED, open-shell LED, HFLD           
  Dmytro Bykov           : pre 5.0 version of the SCF Hessian            
  Marcos Casanova-PÃ¡ez   : Triplet and SCS-CIS(D). UHF-(DLPNO)-IP/EA/STEOM-CCSD. UHF-CVS-IP/STEOM-CCSD   
  Vijay G. Chilkuri      : MRCI spin determinant printing, contributions to CSF-ICE 
  Pauline Colinet        : FMM embedding                                 
  Dipayan Datta          : RHF DLPNO-CCSD density                        
  Achintya Kumar Dutta   : EOM-CC, STEOM-CC                              
  Nicolas Foglia         : Exact transition moments, OPA infrastructure, MCD improvements    
  Dmitry Ganyushin       : Spin-Orbit,Spin-Spin,Magnetic field MRCI      
  Miquel Garcia-Rates    : C-PCM and meta-GGA Hessian, CCSD/C-PCM, Gaussian charge scheme 
  Tiago L. C. Gouveia    : GS-ROHF, GS-ROCIS                             
  Yang Guo               : DLPNO-NEVPT2, F12-NEVPT2, CIM, IAO-localization        
  Andreas Hansen         : Spin unrestricted coupled pair/coupled cluster methods 
  Ingolf Harden          : AUTO-CI MPn and infrastructure                
  Benjamin Helmich-Paris : MC-RPA, TRAH-(SCF,CASSCF), AVAS, COSX integrals, SCF dyn. polar., MC-PDFT, srDFT 
  Lee Huntington         : MR-EOM, pCC                                   
  Robert Izsak           : Overlap fitted RIJCOSX, COSX-SCS-MP3, EOM     
  Riya Kayal             : Wick's Theorem for AUTO-CI, AUTO-CI UHF-CCSDT 
  Emily Kempfer          : AUTO-CI RHF CISDT and CCSDT, approximate NEVPT4
  Christian Kollmar      : KDIIS, OOCD, Brueckner-CCSD(T), CCSD density, CASPT2, CASPT2-K, improved NEVPT2 
  Axel Koslowski         : Symmetry handling                             
  Simone Kossmann        : meta-GGA functionals, TD-DFT gradient, OOMP2, (MP2 Hessian; deprecated post 5.0) 
  Lucas Lang             : DCDCAS, Hyperfine gauge corrections, ICE-SOC+SSC
  Marvin Lechner         : AUTO-CI (C++ implementation), FIC-MRCC        
  Spencer Leger          : CASSCF response                               
  Dagmar Lenk            : GEPOL surface, SMD, ORCA-2-JSON               
  Dimitrios Liakos       : Extrapolation schemes; Compound Job, Property file
  Dimitrios Manganas     : Further ROCIS development; embedding schemes. LFT, Crystal Embedding 
  Dimitrios Pantazis     : SARC Basis sets                               
  Anastasios Papadopoulos: AUTO-CI, single reference methods and gradients 
  Taras Petrenko         : pre 6.0 DFT Hessian and TD-DFT gradient,  ECA, NRVS 
  Petra Pikulova         : Analytic Raman intensities                    
  Peter Pinski           : DLPNO-MP2, DLPNO-MP2 Gradient                 
  Shashank Vittal Rao    : ES-AILFT, MagRelax                            
  Christoph Reimann      : Effective Core Potentials                     
  Marius Retegan         : Local ZFS, SOC                                
  Christoph Riplinger    : Optimizer, TS searches, QM/MM, DLPNO-CCSD(T), (RO)-DLPNO pert. Triples 
  Michael Roemelt        : Original ROCIS implementation, recursive CI coupling coefficients
  Masaaki Saitow         : Open-shell DLPNO-CCSD energy and density      
  Barbara Sandhoefer     : DKH picture change effects                    
  Yorick L. A. Schmerwitz: GMF and freeze-and-release deltaSCF, NEB S-IDPP initial path                    
  Kantharuban Sivalingam : CASSCF convergence/infrastructure, NEVPT2, NEVPT3, NEVPT4(SD), FIC-MRCI and CEPA variants
  Bernardo de Souza      : ESD, SOC TD-DFT                               
  Georgi L. Stoychev     : AutoAux, RI-MP2 NMR, DLPNO-MP2 response, X2C  
  Van Anh Tran           : RI-MP2 g-tensors                              
  Willem Van den Heuvel  : Paramagnetic NMR                              
  Zikuan Wang            : NOTCH, Electric field optimization            
  Frank Wennmohs         : Technical directorship and infrastructure     
  Hang Xu                : AUTO-CI-Response properties                   
                                                                         
[FACCTs GmbH]                                                            
  Markus Bursch, Nicolas Foglia, Miquel Garcia-Rates, Ingolf Harden, Hagen Neugebauer, Anastasios Papadopoulos, 
  Christoph Riplinger, Bernardo de Souza, Georgi L. Stoychev                                                    
                                                                 
  APM, various basis sets, CI-OPT, improved COSX, DLPNO-Multilevel,           
  DOCKER, DRACO, updates on ESD, Fragmentator, GOAT, IRC, LR-CPCM, L-BFGS, MBIS, meta-GGA TD-DFT gradient, ML-optimized integration grids, 
  MM, NACMEs, nearIR, NEB, NEB-TS, NL-DFT gradient (VV10), 2- and 3-layer-ONIOM, interface openCOSMO-RS, QMMM,   
  Crystal-QMMM, RESP, rigid body optimization, SF, symmetry and pop. for TD-DFT, various functionals, SOLVATOR                                   
                                                                         
[Other institutions]                                                     
  V. Asgeirsson          : NEB                                           
  Christoph Bannwarth    : sTDA-DFT, sTD-DFT, PBEh-3c, B97-3c, D3        
  Giovanni Bistoni       : ETS/NOCV, ADLD/ADEX, COVALED                  
  Martin Brehm           : Molecular dynamics                            
  Ronald Cardenas        : ETS/NOCV                                      
  Martina Colucci        : COVALED                                       
  Sebastian Ehlert       : rSCAN, r2SCAN, r2SCAN-3c, D4, dhf basis sets  
  Marvin Friede          : D4 for Fr, Ra, Ac-Lr                          
  Lars Goerigk           : TD-DFT with DH, B97 family of functionals     
  Stefan Grimme          : VdW corrections, initial TS optimization, DFT functionals, gCP, sTDA/sTD-DF 
  Waldemar Hujo          : DFT-NL                                        
  H. Jonsson             : NEB                                           
  Holger Kruse           : gCP                                           
  Marcel Mueller         : wB97X-3c, vDZP basis set                      
  Hagen Neugebauer       : wr2SCAN, Native XTB                           
  Gianluca Regni         : ADLD/ADEX                                     
  Tobias Risthaus        : pre 6.0 range-separated hybrid DFT and stability analysis
  Lukas Wittmann         : regularized MP2, r2SCAN double-hybrids, wr2SCAN        
                                                                         
We gratefully acknowledge several colleagues who have allowed us to      
interface, adapt or use parts of their codes:                            
  Ed Valeev, F. Pavosevic, A. Kumar             : LibInt (2-el integral package), F12 methods 
  Garnet Chan, S. Sharma, J. Yang, R. Olivares  : DMRG                   
  Ulf Ekstrom                                   : XCFun DFT Library      
  Mihaly Kallay                                 : mrcc  (arbitrary order and MRCC methods)
  Frank Weinhold                                : gennbo (NPA and NBO analysis)           
  Simon Mueller                                 : openCOSMO-RS                            
  Christopher J. Cramer and Donald G. Truhlar   : smd solvation model                     
  S Lehtola, MJT Oliveira, MAL Marques          : LibXC Library                           
  Liviu Ungur et al                             : ANISO software                          


 Your calculation uses the libint2 library for the computation of 2-el integrals
 For citations please refer to: http://libint.valeyev.net

 Your ORCA version has been built with support for libXC version: 7.0.0
 For citations please refer to: https://libxc.gitlab.io

 This ORCA versions uses:
   CBLAS   interface :  Fast vector & matrix operations
   LAPACKE interface :  Fast linear algebra routines
   SCALAPACK package :  Parallel linear algebra routines
   Shared memory     :  Shared parallel matrices
   BLAS/LAPACK       :  OpenBLAS 0.3.29  USE64BITINT DYNAMIC_ARCH NO_AFFINITY Haswell SINGLE_THREADED
        Core in use  :  Haswell
   Copyright (c) 2011-2014, The OpenBLAS Project


  ***********************************
  * Starting time: Fri Sep 26 20:23:19 2025
  * Host name:     Host-001
  * Process ID:    33453
  * Working dir.:  /home/ppravatto/Documenti/Esperimenti/ChimicaComputazionale/SPyCCI/water_29kdqzma_orca_PBU_def2-SVP_vacuum_spe
  ***********************************

NOTE: MaxCore=750 MB was set to SCF,MP2,MDCI,CIPSI,MRCI,RASCI and CIS
      => If you want to overwrite this, your respective input block should be placed after the MaxCore statement

            !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
                                 INPUT ERROR
            UNRECOGNIZED OR DUPLICATED KEYWORD(S) IN SIMPLE INPUT LINE
              PBU                    
            !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
[file orca_main/main_input_keywordline.cpp, line 13872]: 

